linear_vel_1: 1.0
linear_vel_2: 1.5
angular_vel_1: -2.0
angular_vel_2: -2.5
linear_time: 3.0
angular_time: 1.0
